common/test/acceptance/tests/helpers.py

```diff
@@ -167,6 +168,31 @@ def enable_css_animations(page):
     """)
 
 
+def generate_course_key(org, number, run):
+    """
+    Makes a CourseLocator from org, number and run
+    """
+    default_store = os.environ.get('DEFAULT_STORE', 'draft')
+    return CourseLocator(org, number, run, deprecated=(default_store == 'draft'))
+
+
+def select_ddl_by_value(browser_query, value):
+    """
+    Selects a html select element by matching value attribute
```

Comment:
You inspired me to create some Select helper methods in my own PR. We should try to keep method names consistent. So "select_option_by_value" and "is_option_value_selected"?

https://github.com/edx/edx-platform/pull/6308/files#diff-d2fd85d338b26adbce4621586d16300dR173